enterprise/server/remote_execution/containers/ociruntime/ociruntime.go

```diff
@@ -1128,6 +1135,14 @@ func (c *ociContainer) invokeRuntime(ctx context.Context, command *repb.Command,
 		runError = nil
 	}
 	code, err := commandutil.ExitCode(ctx, cmd, runError)
+
+	// Some actions are prone to SIGSEGV when running on an executor that is close to its memory limits.
+	// Return a retryable error so we can make sure that the failure is not infrastructure related.
+	if code == ociSIGSEGVExitCode {
+		log.CtxWarning(ctx, "action exited with SIGSEGV")
+		err = errSIGSEGV
```

Comment:
nit: not sure exactly what happens if we forget this, but I think usually we want either `result.ExitCode>=0` or `result.Error!=nil`, but not both

```suggestion
		code = commandutil.NoExitCode
		err = errSIGSEGV
```